data/C4DT/projects.yaml

```diff
@@ -19,6 +19,51 @@ projects:
     tags:
       - Access Control
       - Development
+    incubator:
+      type: retired
+      work: 2023/Q3-2024/Q2 Testing ad implementing the demonstrator
```

Comment:
Testing **and** implementing

Reply:
Thanks. That's my crappy keyboard at home. I really need to replace it!